sklearn/linear_model/_ridge.py

```diff
@@ -767,7 +767,12 @@ def fit(self, X, y, sample_weight=None):
 class RidgeClassifier(LinearClassifierMixin, _BaseRidge):
     """Classifier using Ridge regression.
 
-    Read more in the :ref:`User Guide <ridge_regression>`.
+    This classifier first converts binary targets to ``{-1, 1}`` and then
+    treats the problem as a regression task with a
+    :class:`~sklearn.linear_model.Ridge` regressor.
+
+    Read more about Ridge Regression in the :ref:`User Guide
```

Comment:
the original version seems enough and more consistent

Reply:
You mean that this line should be replaced by 

    Read more in the :ref:`User Guide <ridge_regression>`.

But the additional paragraph "This classifier first converts..." should be kept, right?

Reply:
I added something in-between.